bin/jurassic-tube-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+echo "Checking if ${PWD}/docker/bin/jt directory exists..."
+
+if [ -d "${PWD}/docker/bin/jt" ]; then
+    echo "${PWD}/docker/bin/jt already exists."
+else
+    echo "Creating ${PWD}/docker/bin/jt directory..."
+    mkdir -p "${PWD}/docker/bin/jt"
+fi
+
+echo "Downloading the latest version of the installer script..."
+echo
+
+# Download the installer (if it's not already present):
+if [ ! -f "${PWD}/docker/bin/jt/installer.sh" ]; then
+    # Download the installer script:
+    curl "https://jurassic.tube/get-installer.php?env=wcpay" -o ${PWD}/docker/bin/jt/installer.sh && chmod +x ${PWD}/docker/bin/jt/installer.sh
```

Comment:
```suggestion
    curl "https://jurassic.tube/get-installer.php?env=wc-stripe" -o ${PWD}/docker/bin/jt/installer.sh && chmod +x ${PWD}/docker/bin/jt/installer.sh
```

Hi @diegocurbelo, thanks for doing this!
I adjusted Jurassic Tube to support the Stripe environment, that way it'll display the correct commands in its output: `yarn jt:start` and `yarn jt:stop`.